jvb/src/main/kotlin/org/jitsi/videobridge/cc/allocation/BitrateController.kt

```diff
@@ -200,11 +200,7 @@ class BitrateController<T : MediaSourceContainer> @JvmOverloads constructor(
         val nowMs = clock.instant().toEpochMilli()
         val allocation = bandwidthAllocator.allocation
         allocation.allocations.forEach { singleAllocation ->
-            val allocationTargetBitrate: Bandwidth? = if (config.useVlaTargetBitrate) {
-                singleAllocation.targetLayer?.targetBitrate ?: singleAllocation.targetLayer?.getBitrate(nowMs)
-            } else {
-                singleAllocation.targetLayer?.getBitrate(nowMs)
-            }
+            val allocationTargetBitrate: Bandwidth? = singleAllocation.targetLayer?.getBitrate(nowMs)
```

Comment:
This will cause probing to happen, even when we're fully sending ideal, if the VLA bitrate is greater than the measured bitrate, which seems bad.
Conversely if the VLA bitrate is less than the measured bitrate we won't probe up, though I'm confused if that will matter.